### ISSUE TITLE
Altitude is logged but not used

### ISSUE THREAD

**Issue by AdamLeonSmith** [tripleblindmarket/covid-safe-paths#759]
Data that is not used/required should not be collected for privacy purposes.  It appears that altitude is logged in the internal application database but not used as part of the contact tracing flow.

**Comment by summetj** [comment]
I can see that potentially being useful (e.g. floors within a building) but if we are not exporting it as part of the json file there is no reason to store it locally.  Suggest you ask around on the slack #appdev channel to make sure there are no future plans for this.....

**Comment by AdamLeonSmith** [comment]
I disagree this is low priority as it is a privacy issue, and we are supposed to be privacy focussed.  If we plan to use this in the future, that's fine, we can document it.  If we don't, it is an easy fix to remove, I would be willing to submit a PR for.  Can anyone comment on this?

**Comment by Ferrumofomega** [comment]
It is important to address from a privacy perspective @AdamLeonSmith. Would you mind submitting a PR for this change?
